ceph.spec.in

```diff
@@ -102,6 +102,18 @@ BuildRequires:	checkpolicy
 BuildRequires:	selinux-policy-devel
 BuildRequires:	/usr/share/selinux/devel/policyhelp
 %endif
+%if 0%{with make_check}
+%if 0%{?fedora} || 0%{?rhel}
+BuildRequires:	python-cherrypy
+BuildRequires:	python-werkzeug
```

Comment:
This just moves the build-time dependencies, the runtime dependencies are still in place. I would like to see a full jenkins builder job for this though.